### ISSUE TITLE
Add field "type" 

### ISSUE THREAD

**Issue by trugwaldsaenger** [hbz/oerworldmap#1622]
JRC proposes in their report "[Going Open - Policy Recommendations on Open Education in Europe](https://oerworldmap.org/resource/urn%3Auuid%3A6b49470c-bdd3-4b75-aad7-415b05f31acc?)" to distinguish four types of policies:
![grafik](https://user-images.githubusercontent.com/6639965/43640122-f131d724-971e-11e8-8224-73d13769b49f.png)

The dinstinction as such seems well funded to me, but we have to find a shorter wording. I propose following:

- Q1: dedicated OE/OER policy
- Q2: ICT policy with open component
- Q3: Educatiional policy with open component
- Q4: Open government policy with educational component

**Comment by karindr** [comment]
Two suggestions:

1. change " ICT policy with open component" to " ICT policy with open educational component" because in ITC open alone could also be something like "open source"

2. use "OE component" in Q2, Q3, and Q4.

I would prefer the second because in the context of OE/OER policies the term "OE" should be known

**Comment by trugwaldsaenger** [comment]
@karindr: Good point - you are right that the JRC typology/approach is broader than ours. I will discuss this issue with JRC on occassion. For the beginning I propose that we stay with their wording.

**Comment by trugwaldsaenger** [comment]
@karindr: If we continue working on the table I propose that we add a new column for this field and check, how it works for the data currently in the registry.

**Comment by trugwaldsaenger** [comment]
A missing fifth type would be 
Q5: Labour market policy with open component

**Comment by trugwaldsaenger** [comment]
@programmieraffe: Does this makes sense to you?

**Comment by trugwaldsaenger** [comment]
Just refined the needed value set for the field "type":
   - Dedicated OE/OER policy
   - ICT policy with OE/OER component
   - Educational policy with OE/OER component
   - Openness policy with educational component
   - Labour market policy with OE/OER component
@literarymachine: from my point of view ready to implement

**Comment by trugwaldsaenger** [comment]
Looks good already quite good. Data input works fine, but it seems to me that the field is not yet published on the profile. As far as the name of the field is concerned I will think about an alternative later this evening.

**Comment by literarymachine** [comment]
> Data input works fine, but it seems to me that the field is not yet published on the profile.

For now, the field follows the "second level type logic". I did this so that we have some more structure for the policies in the `Find` menu:

![image](https://user-images.githubusercontent.com/149825/55495039-596f3e80-563c-11e9-9479-9afcb2bcc712.png)

As with all second level types, it is currently displayed on the very top:

![image](https://user-images.githubusercontent.com/149825/55495127-8885b000-563c-11e9-8fdf-a30d55234a6c.png)

I agree that this is very hidden, we should probably repeat that information (in the sidebar?) for all second level types anyways.
